public/stylesheets/site/2.0/11-group-listbox.css

```diff
@@ -1,5 +1,5 @@
 /*==GROUP: LISTBOX 
-http://otwcode.github.com/docs/front_end_coding/patterns/listbox
+https://otwcode.github.io/docs/front_end_coding/patterns-listbox
```

Comment:
The `-` should be a `/` in these next few files as well. (For some reason, `pattern-index` is the only one that actually uses a `-`. 🤷 )